docs/docsite/rst/user_guide/become.rst

```diff
@@ -749,6 +749,13 @@ Limitations of become on Windows
 
 * The Secondary Logon service ``seclogon`` must be running to use ``ansible_become_method: runas``
 
+Resolving Temporary File Error Messsages
+----------------------------------------
+
+"Failed to set permissions on the temporary files Ansible needs to create when becoming an unprivileged user"
```

Comment:
This will support people googling for exactly this error message and having (potentially) an immediately resolvable result